content/dota_addons/legion_td/panorama/layout/custom_game/main_hud.xml

```diff
@@ -4,13 +4,36 @@
         <include src="file://{resources}/styles/custom_game/main_hud.css" />
         <include src="file://{resources}/styles/custom_game/statistics.css" />
         <include src="file://{resources}/styles/custom_game/shop.css" />
+		<include src="file://{resources}/styles/custom_game/popup_window.css" />
     </styles>
     <scripts>
         <include src="file://{resources}/scripts/custom_game/utils.js" />
         <include src="file://{resources}/scripts/custom_game/main_hud.js" />
     </scripts>
     <Panel hittest="false" class="Hud">
         <Panel hittest="false" class="DontKnow">
+		
+			<Panel hittest="false" id="PopupWindow">
+				<Panel id="BackgroundImage"/>
+				<Panel id="Contents">
+					<Label id="Title" text="#SPRING_CLEANING"/>
+					<Label id="Message" text="#Call_To_Arms_Description"/>
+				</Panel>
+				<Label id="VersionNumber" text="#Call_to_Arms"/>
+				<Button id="DiscordButton"  onactivate="ExternalBrowserGoToURL('https://discord.gg/pmMnfdA');" onmouseover="UIShowTextTooltip(#Ad_updated)"  onmouseout="UIHideTextTooltip()">
+					<Image id="DiscordImage" src="file://{images}/custom_game/game_info/Discord.png" />
+				</Button>
+				<Panel id="Buttons" class="PopupButtonRow">
+					<TextButton class="ButtonBevel" text="#DOTA_Continue" onactivate="OnContinueButtonPressed()" />
+				</Panel>
+			</Panel>
+	
+			<Panel id="StyleButtons" hittest="false">
+				<Button class="ButtonBevel" id="NextButton" onactivate="OnCleaningButtonPressed()" onload="DOTAShowTextTooltip('#Announcement')" onmouseover="UIShowTextTooltip(#Announcement)"  onmouseout="UIHideTextTooltip()">
+					<Label id="Style2ButtonLabel" text="#SPRING_CLEANING"/>
+				</Button>
+			</Panel>
```

Comment:
This still shows up every game right? Could you move it to the loading screen instead? So it a) isn't popping up every game and b) isn't happens earlier in the game

Reply:
It is hidden at the start, gets open  when someone clicks to a button. 

![unknown](https://user-images.githubusercontent.com/30335579/72209284-b4806c80-34bd-11ea-9845-cf6f45bf0027.png)

There is no possible place on loading screen and if I put it as extra, it will cover whole screen and will include many